app/services/appointment_scheduler.rb

```diff
@@ -0,0 +1,49 @@
+class AppointmentScheduler
+  def initialize(max_schedule_time_ahead:)
+    @max_schedule_time_ahead = if max_schedule_time_ahead.is_a?(ActiveSupport::Duration)
+      max_schedule_time_ahead
+    else
+      max_schedule_time_ahead.days
```

Comment:
se é só a gente que consome o serviço pq temos esse tratamento?

Reply:
Programação defensiva :nervoso:

Achei que faria sentido suportar os dois, just in case...

Reply:
hahaha tá se defendendo de si mesmo no futuro?